OOP/OOP6_ResizableArray/src/test/java/LukCharListTest.java

```diff
@@ -0,0 +1,62 @@
+import static org.junit.Assert.*;
+
+public class LukCharListTest {
+
+    @org.junit.Test
+    public void save() {
+        //given
+        final LukCharList lukCharList = new LukCharList();
+        final int expectedIntPosition = 0;
+        //when
+        final int actualIntPosition = lukCharList.save('5');
+        //then
+        assertEquals(actualIntPosition,expectedIntPosition);
```

Comment:
Since this test only tests if method "save" returns correct index, it should be performed on larger number of invocations (e.g. 100).